### PR TITLE
Refs unification

### PR DESCRIPTION
**Pull request by alirezazolanvari**
#### Reference Issues/PRs
#173 

#### What does this implement/fix? Explain your changes.
The references are unified with IEEE format

#### Any other comments?
Please carefully compare the new references' titles with the previous ones. Be sure that the formats are correct.

### CODE REVIEW

**Comment by sepandhaghighi** [comment]
@alirezazolanvari Thanks for your efforts 💯 
I will review this PR in the next few days.
But as a side note, I think it would better if we mention `IEEE` format somewhere in `CONTRIBUTING.md`

File: README.md [hunk 1]
```diff
@@ -738,156 +738,155 @@ or send an email to [info@pycm.ir](mailto:info@pycm.ir "info@pycm.ir").
...
 
-<blockquote>32- V. Sindhwani, P. Bhattacharya, and S. Rakshit, "Information theoretic feature crediting in multiclass support vector machines," in Proceedings of the 2001 SIAM International Conference on Data Mining, pp. 1-18, 2001.</blockquote> 
+<blockquote>33- M. Bekkar, H. K. Djemaa, and T. A. Alitouche, "Evaluation measures for models assessment over imbalanced datasets," <i>J Inf Eng Appl</i>, vol. 3, no. 10, 2013.</blockquote>
 
-<blockquote>33- M. Bekkar, H. K. Djemaa, and T. A. J. J. I. E. A. Alitouche, "Evaluation measures for models assessment over imbalanced data sets," in Journal of Information Engineering and Applications, 2013.</blockquote>
+<blockquote>34- W. J. Youden, "Index for rating diagnostic tests," <i>Cancer</i>, vol. 3, no. 1, pp. 32-35, 1950.</blockquote>
 
-<blockquote>34- W. J. J. C. Youden, "Index for rating diagnostic tests," in Cancer, pp. 32-35, 1950.</blockquote>
+<blockquote>35- S. Brin, R. Motwani, J. D. Ullman, and S. Tsur, "Dynamic itemset counting and implication rules for market basket data," in <i>Proceedings of the 1997 ACM SIGMOD international conference on Management of data</i>, 1997, pp. 255-264.</blockquote>
```
Comment (sadrasabouri) [inline README.md:809]: another blank space here :point_up:

File: README.md [hunk 2]
```diff
@@ -738,156 +738,155 @@ or send an email to [info@pycm.ir](mailto:info@pycm.ir "info@pycm.ir").
...
+<blockquote>35- S. Brin, R. Motwani, J. D. Ullman, and S. Tsur, "Dynamic itemset counting and implication rules for market basket data," in <i>Proceedings of the 1997 ACM SIGMOD international conference on Management of data</i>, 1997, pp. 255-264.</blockquote> 
 
-<blockquote>35- S. Brin, R. Motwani, J. D. Ullman, and S. J. A. S. R. Tsur, "Dynamic itemset counting and implication rules for market basket data," in Proceedings of the 1997 ACM SIGMOD international conference on Management of datavol, pp. 255-264, 1997.</blockquote> 
+<blockquote>36- S. Raschka, "MLxtend: Providing machine learning and data science utilities and extensions to Python's scientific computing stack," <i>Journal of open source software</i>, vol. 3, no. 24, p. 638, 2018.</blockquote> 
 
-<blockquote>36- S. J. T. J. o. O. S. S. Raschka, "MLxtend: Providing machine learning and data science utilities and extensions to Python’s scientific computing stack," in Journal of Open Source Software, 2018.</blockquote> 
+<blockquote>37- J. Bray and J. Curtis, "An ordination of upland forest communities of southern Wisconsin. Ecological Monographs (27)," ed: PRIMER-E Plymouth, 1957, pp. 325-349.</blockquote>
```
Comment (sadrasabouri) [inline README.md:813]: another blank space here :point_up:

File: README.md [hunk 3]
```diff
@@ -738,156 +738,155 @@ or send an email to [info@pycm.ir](mailto:info@pycm.ir "info@pycm.ir").
...
+<blockquote>36- S. Raschka, "MLxtend: Providing machine learning and data science utilities and extensions to Python's scientific computing stack," <i>Journal of open source software</i>, vol. 3, no. 24, p. 638, 2018.</blockquote> 
 
-<blockquote>36- S. J. T. J. o. O. S. S. Raschka, "MLxtend: Providing machine learning and data science utilities and extensions to Python’s scientific computing stack," in Journal of Open Source Software, 2018.</blockquote> 
+<blockquote>37- J. Bray and J. Curtis, "An ordination of upland forest communities of southern Wisconsin. Ecological Monographs (27)," ed: PRIMER-E Plymouth, 1957, pp. 325-349.</blockquote>	
 
-<blockquote>37- J. BRAy and J. CuRTIS, "An ordination of upland forest communities of southern Wisconsin.-ecological Monographs," in journal of Ecological Monographs, 1957.</blockquote>	
+<blockquote>38- J. L. Fleiss, J. Cohen, and B. S. Everitt, "Large sample standard errors of kappa and weighted kappa," <i>Psychological bulletin</i>, vol. 72, no. 5, p. 323, 1969.</blockquote>
```
Comment (sadrasabouri) [inline README.md:815]: another blank space here :point_up:

File: README.md [hunk 4]
```diff
@@ -738,156 +738,155 @@ or send an email to [info@pycm.ir](mailto:info@pycm.ir "info@pycm.ir").
...
 
-<blockquote>37- J. BRAy and J. CuRTIS, "An ordination of upland forest communities of southern Wisconsin.-ecological Monographs," in journal of Ecological Monographs, 1957.</blockquote>	
+<blockquote>38- J. L. Fleiss, J. Cohen, and B. S. Everitt, "Large sample standard errors of kappa and weighted kappa," <i>Psychological bulletin</i>, vol. 72, no. 5, p. 323, 1969.</blockquote> 	
 
-<blockquote>38- J. L. Fleiss, J. Cohen, and B. S. J. P. B. Everitt, "Large sample standard errors of kappa and weighted kappa," in Psychological Bulletin, p. 323, 1969.</blockquote> 	
+<blockquote>39- M. Felkin, "Comparing classification results between n-ary and binary problems," in <i>Quality Measures in Data Mining</i>: Springer, 2007, pp. 277-301.</blockquote> 
 
-<blockquote>39- M. Felkin, "Comparing classification results between n-ary and binary problems," in Quality Measures in Data Mining: Springer, pp. 277-301, 2007.</blockquote> 
+<blockquote>40- R. Ranawana and V. Palade, "Optimized precision-a new measure for classifier performance evaluation," in <i>2006 IEEE International Conference on Evolutionary Computation</i>, 2006: IEEE, pp. 2254-2261.</blockquote>
```
Comment (sadrasabouri) [inline README.md:819]: another white space here :point_up:

**Review by sadrasabouri — CHANGES_REQUESTED** [review]
Thank you @alirezazolanvari :1st_place_medal: 
There were tiny things that I mentioned in comments, also there were some changes in date of each references that I decided not to mention them (due to make the review lighter) but there should be reasonable reason why these changes are necessary.

**Comment by sadrasabouri** [comment]
> @alirezazolanvari Thanks for your efforts 100
> I will review this PR in the next few days.
> But as a side note, I think it would better if we mention `IEEE` format somewhere in `CONTRIBUTING.md`

Also logging this change in `CHANGELOG.md` may be helpful.

File: README.md [hunk 5]
```diff
@@ -738,156 +738,155 @@ or send an email to [info@pycm.ir](mailto:info@pycm.ir "info@pycm.ir").
...
+<blockquote>17- J.-M. Wei, X.-J. Yuan, Q.-H. Hu, and S.-Q. Wang, "A novel measure for evaluating classifiers," <i>Expert Systems with Applications</i>, vol. 37, no. 5, pp. 3799-3809, 2010.</blockquote>
 
-<blockquote>17- J.-M. Wei, X.-J. Yuan, Q.-H. Hu, and S.-Q. J. E. S. w. A. Wang, "A novel measure for evaluating classifiers," in Expert Systems with Applications, pp. 3799-3809, 2010.</blockquote>
+<blockquote>18- I. Kononenko and I. Bratko, "Information-based evaluation criterion for classifier's performance," <i>Machine learning</i>, vol. 6, no. 1, pp. 67-80, 1991.</blockquote>
 
-<blockquote>18- I. Kononenko and I. J. M. L. Bratko, "Information-based evaluation criterion for classifier's performance," in Machine Learning, pp. 67-80, 1991.</blockquote>
+<blockquote>19- R. Delgado and J. D. Núnez-González, "Enhancing confusion entropy as measure for evaluating classifiers," in <i>The 13th International Conference on Soft Computing Models in Industrial and Environmental Applications</i>, 2018: Springer, pp. 79-89.</blockquote>
```
[the remark this answers is not in the document]
Comment (alirezazolanvari) [reply README.md:777]: In IEEE format for citing conference papers, `in` should be written before the conference's name.

**Review by sadrasabouri — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -738,156 +738,155 @@ or send an email to [info@pycm.ir](mailto:info@pycm.ir "info@pycm.ir").
...
+<blockquote>53- Y. Otsuka, "The faunal character ofthe Japanese Pleistocene marine Mollusca, as evidence of climate having become colder during the Pleistocene in Japan," <i>Biogeograph Soc Japan</i>, vol. 6, no. 16, pp. 165-70, 1936.</blockquote>
 
-<blockquote>54- A. Tversky, "Features of similarity," Psychological review, vol. 84, no. 4, p. 327, 1977.</blockquote>
+<blockquote>54- A. Tversky, "Features of similarity," <i>Psychological review</i>, vol. 84, no. 4, p. 327, 1977.</blockquote>
 
-<blockquote>55- K. Boyd, K. H. Eng, and C. D. Page, "Area under the precision-recall curve: point estimates and confidence intervals," in Joint European conference on machine learning and knowledge discovery in databases, 2013, pp. 451-466: Springer.</blockquote>
+<blockquote>55- K. Boyd, K. H. Eng, and C. D. Page, "Area under the precision-recall curve: point estimates and confidence intervals," in <i>Joint European conferenceon machine learning and knowledge discovery in databases</i>, 2013: Springer, pp. 451-466.</blockquote>
```
Comment (sepandhaghighi) [inline README.md:849]: `conferenceon`?

File: README.md [hunk 7]
```diff
@@ -738,156 +738,155 @@ or send an email to [info@pycm.ir](mailto:info@pycm.ir "info@pycm.ir").
...
 
-<blockquote>56- J. Davis and M. Goadrich, "The relationship between Precision-Recall and ROC curves," in Proceedings of the 23rd international conference on Machine learning, 2006, pp. 233-240: ACM.</blockquote>
+<blockquote>56- J. Davis and M. Goadrich, "The relationship between Precision-Recall and ROC curves," in <i>Proceedings of the 23rd international conference on Machine learning</i>, 2006, pp. 233-240.</blockquote>
 
-<blockquote>57- M. Kuhn, "Building predictive models in R using the caret package," Journal of statistical software, vol. 28, no. 5, pp. 1-26, 2008.</blockquote>
+<blockquote>57- M. Kuhn, "Building predictive models in R using the caret package," <i>J Stat Softw</i>, vol. 28, no. 5, pp. 1-26, 2008.</blockquote>
```
Comment (sepandhaghighi) [inline README.md:853]: Journal name is `Journal of statistical software`

https://www.jstatsoft.org/article/view/v028i05

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

**Comment by sepandhaghighi** [comment]
@alirezazolanvari 
Please take a look at my comments, then I will update `CHANGELOG.md`, `CONTRIBUTING.md` and `Document.ipynb`

**Review by sepandhaghighi — APPROVED** [review]

**Comment by sepandhaghighi** [comment]
LGTM 👍 
@sadrasabouri @alirezazolanvari Please take a look at my commits (https://github.com/sepandhaghighi/pycm/pull/362/commits/0bf28f46cec4599659e624c5074e57bd2de5090a, https://github.com/sepandhaghighi/pycm/pull/362/commits/cd416c1f14c8432aeb957b149d0d5be204e80b7c, https://github.com/sepandhaghighi/pycm/pull/362/commits/ad546700702c24906f4e1dee03133bfbaaa2fc00)
If everything is okay, please merge this `PR` after approval.

**Review by sadrasabouri — APPROVED** [review]
LGTM

**Pull request merged**